students/chan/westagram/users/views.py

```diff
@@ -42,9 +48,12 @@ def post(self, request):
             email    = data['email']
             password = data['password']
 
-            if not User.objects.filter(email=email, password=password).exists():
+            if not User.objects.filter(email=email).exists():
                 return JsonResponse({'Message' : 'INVALID_USER'}, status=401)
 
+            if not bcrypt.checkpw(password.encode('utf-8'),User.objects.filter(email=email).get().password.encode('utf-8')):
+                return JsonResponse({'Message' : 'INVALID_PASSWORD'}, status=401)
```

Comment:
email이 틀렸을 경우와 비민번호가 틀렸을 경우 같은 메시지를 보내는 것이 보안상 더 좋습니다.
```suggestion
                return JsonResponse({'Message' : 'INVALID_USER'}, status=401)
```

Reply:
1. 요청하신 가독성 향상을 위한 코드 간략화 진행하였습니다.
2. 보안을 위하여 email 비밀번호 틀렸을 경우 같은 메세지 보내기위해
 Message : INVALID_USER 로 통일 하였습니다.